src/chainstate/stacks/block.rs

```diff
@@ -565,8 +565,35 @@ impl StacksBlock {
         }
     }
 
+    /// Verify that all transactions are supported in the current epoch
+    pub fn validate_transactions_static_epoch(
+        txs: &Vec<StacksTransaction>,
```

Comment:
In most cases, it's better to use `&[T]` instead of `&Vec<T>` in arguments, since `Vec` will autoref to that anyways, and then you have more freedom to use slices in tests, etc.

Reply:
Done